README.md

```diff
@@ -2,13 +2,13 @@
 
 ## Purpose
 
-The purpose of this Nextflow workflow is to automate the process of staging Synapse files in an accessible location (_e.g._ an S3 bucket). In turn, these staged files can be used as input for a general-purpose (_e.g._ nf-core) workflow that doesn't contain Synapse-specific steps for downloading data. This workflow is intended to be run first in preparation for other data processing workflows.
+The purpose of this Nextflow workflow is to automate the process of staging Synapse and Seven Bridges CGC files in an accessible location (_e.g._ an S3 bucket). In turn, these staged files can be used as input for a general-purpose (_e.g._ nf-core) workflow that doesn't contain platform-specific steps for downloading data. This workflow is intended to be run first in preparation for other data processing workflows.
```

Comment:
You refer to the CGC here and in a few other places below. Since you support the `SB_API_ENDPOINT` environment variable, this workflow can technically work with any SevenBridges API, including Cavatica. So, I would just search-and-replace any instance of "SevenBridges CGC" to just "SevenBridges".